ons_alpha/jinja2/templates/components/streamfield/corrections_block.html

```diff
@@ -1,5 +1,8 @@
-<h3>{{ _("Corrections") }}</h3>
+{% from "components/details/_macro.njk" import onsDetails %}
 
-{% for item in value %}
-    {{ item }}
-{% endfor %}
+
+{% call onsDetails({"id": "correction-v" + version_num|string, "title": value.when|date("DATETIME_FORMAT")}) %}
```

Comment:
I did mean wrapping the whole `ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html` in a details block, not individual corrections. This is the pattern currently used

Reply:
Oh, yes that's much closer to how it is currently - not sure how I missed that!